Add instructions on how to halt shardeum network and obtain SHM (#18)
shardeum/shardeum: README.md

For usage instructions and available options for the shardus command-line tool, run `shardus --help`

## Stopping the Running Network and Cleanup

```bash
shardus stop && shardus clean && rm -rf instances
```

> This command will stop the running network, cleans up any associated resources, and removes instances from the system.

Reviewer: ```suggestion
This command will stop the running network, clean up the associated resources, and remove the `instances` folder from the system.
```

### Obtaining Test Tokens on the Local Network

> If you're running a local Shardeum network, navigate to the [src/config/genesis.json](src/config/genesis.json) file within the shardeum repository. Simply add your wallet address to this file, then restart the network. Voila! You'll now have SHM tokens available on your wallet within the local network.

Reviewer: ```suggestion
If you're running a local Shardeum network, add your wallet address to the [src/config/genesis.json](src/config/genesis.json) file and restart the network. Voila! You'll now have SHM tokens available on your wallet within the local network.
```
